WTWSMS/history/characters/roman.txt

```diff
@@ -84,7 +84,7 @@
 }
 20783 = {
 	name="Manlius Boethius" # Praetorian Prefect
-	#dynasty=1042312
+	dynasty=1042285
```

Comment:
This may have been disabled for a reason (I don't know what reason, though).

Reply:
They previously used Anicius dynasty name. I think it was disabled because they not connected with Anicians.

Reply:
Yes, they aren't connected.

Reply:
If they belong to the same dynasty, I think it's better to give it to them even if they aren't connected.

Reply:
That breaks the dynasty though.

Reply:
Why? Disconnected parts of one dynasty are handled normally by the game, they just appear without a link in the dynasty tree.

Reply:
Ok, fine if it doesn't.